src/model/comment-model.js

```diff
@@ -11,4 +11,5 @@ const getCommentData = (item) => ({
 export default class CommentsModel {
   comments = getCommentsListMock();
   getCommentsList = () => this.comments.map((value) => getCommentData(value));
+  getCommentsById = (arrayOfId) => this.comments.filter((element) => arrayOfId.includes(element.id));
```

Comment:
а ведь с точки зрения нормализованных данных правильнее этот api сделать 
```suggestion
  getFilmComments = (filmId) => ....
```

и возникает вопрос. Зачем два разных объекта - модель фильмов и модель комментов?